### PR TITLE
feat(pci.projects.new): manage polling during project creation

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/creating/creating.controller.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    })
+      .then(() => this.onProjectCreated());
+  }
+
+  /* =====================================
+  =            Initialization            =
+  ====================================== */
+
+  $onInit() {
+    this.Poller.kill({
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/creating/creating.controller.js]: We used to kill poller even on `$onDestroy` hook.

Is that make sense here too?

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/projects/project/creating/creating.controller.js]: Isn't that a `Promise` ?

File: packages/manager/modules/pci/src/projects/project/creating/creating.controller.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+export default class ProjectCreatingCtrl {
+  /* @ngInject */
+  constructor(Poller) {
+    this.Poller = Poller;
+  }
+
+  startCreationPolling() {
+    return this.Poller.poll(`/cloud/project/${this.projectId}`, null, {
+      namespace: 'pci.projects.creating',
+      successRule(details) {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/creating/creating.controller.js]:
```suggestion
      successRule(details) => details.status === 'ok'
```

File: packages/manager/modules/pci/src/projects/project/project.routing.js [hunk 3]
```diff
@@ -16,6 +16,17 @@ export default /* @ngInject */ ($stateProvider) => {
         format: 'json',
         value: ['.'],
       },
+      redirectTo: ($transitions) => {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/project.routing.js]:
```suggestion
      redirectTo: /* @ngInject */ ($transitions) => $transitions.injector()
        .getAsync('project')
        .then((project) => project.status === 'creating'
          ? 'pci.projects.project.creating'
          : null
```

Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/project/project.routing.js]: Nop, redirectTo should not be injected. Please have a look in uiRouter documentation :)

**Pull request merged**